python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/docker-compose.yml

```diff
@@ -0,0 +1,212 @@
+# Airbyte docker-compose file
+# Sourced from https://github.com/airbytehq/airbyte/blob/master/docker-compose.yml
```

Comment:
Wowza - is this the minimal necessary config? Or just a direct copy? Non blocking, but it'd be nice if we were able to pare this down to make it more obvious which levers we're actually pulling and what we're relying on the defaults for.

Reply:
We may be able to pare it down more, or to just directly pull this file from their GitHub repo instead. We could probably just pull this file and the env file in particular.